components/form/CheckboxGroup.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import PropTypes from 'prop-types';
+import { CheckboxLabel, ErrorLabel } from '.';
+
+export function Checkbox({ ...props }) {
+  const { register } = useFormContext();
+  return (
+    <>
+      <div className={`flex items-center ${props.className}`}>
+        <input
+          id={`checkbox-${props.value}`}
+          className="w-4 h-4 text-amcovad-primary-400 border-gray-200 rounded focus:ring-amcovad-primary-300 "
+          name={props.name}
+          type="checkbox"
+          {...register(props.name)}
+          value={props.value}
+        />
+        <CheckboxLabel htmlFor={`checkbox-${props.value}`} label={props.label} />
+      </div>
+    </>
+  );
+}
+export function CheckboxGroup({ name, label, value }) {
```

Comment:
This is not needed and should be deleted.

This should contain the function you used in `CheckboxGroups`. If we need a single Checkbox, we can pass an array with a single value into the CheckboxGroup.

Reply:
you, re  very right  reason I left here, is due to the link `Link` inside the label which as color style and it can be passed inside an array to implement the purpose according to the design guild
- Subject to be review